pennylane/data/data_manager/__init__.py

```diff
@@ -361,110 +356,54 @@ def load(  # pylint: disable=too-many-arguments
     return [Dataset.open(path, "a") for path in download_paths]
 
 
-def list_datasets() -> dict:
-    r"""Returns a dictionary of the available datasets.
-
-    Return:
-        dict: Nested dictionary representing the directory structure of the hosted datasets.
-
-    .. seealso:: :func:`~.load_interactive`, :func:`~.list_attributes`, :func:`~.load`.
-
-    **Example:**
-
-    Note that the results of calling this function may differ from this example as more datasets
-    are added. For updates on available data see the `datasets website <https://pennylane.ai/datasets>`_.
+def _interactive_request_data_name(data_names):
+    """Prompt the user to select a data name."""
+    print("Please select the data name from the following:")
+    for i, option in enumerate(data_names):
+        print(f"{i + 1}: {option}")
+    choice = input("Choice of data name: ")
```

Comment:
```suggestion
    choice = input("Choice of data name: ").strip()
```
Just in case the user accidentally typed an extra space or something.